web/src/pages/404.js

```diff
@@ -1,63 +1,89 @@
 import * as React from 'react';
-import { Link } from 'gatsby';
+import { graphql, Link } from 'gatsby';
 import styled from 'styled-components';
+import Img from 'gatsby-image';
 import tw from 'twin.macro';
+import PropTypes from 'prop-types';
 
-const Main = styled.main`
-    ${tw`container`}
+import Layout from '../page-layouts/layout';
+import Hero from '../components/hero';
+import CtaSection from '../components/ctaSection';
 
-    color: '#232129',
-    fontFamily: '-apple-system, Roboto, sans-serif, serif',
+const Wrapper = styled.div`
+    ${tw`flex flex-col justify-center items-center content-center`}
 `;
 
-// styles
-// const pageStyles = {
-//     color: '#232129',
-//     fontFamily: '-apple-system, Roboto, sans-serif, serif',
-// };
-const headingStyles = {
-    marginTop: 0,
-    marginBottom: 64,
-    maxWidth: 320,
-};
+const StyledHeading = styled.h2`
+    ${tw`font-bold text-center text-5xl w-2/4 mb-8`}
 
-const paragraphStyles = {
-    marginBottom: 48,
-};
-const codeStyles = {
-    color: '#8A6534',
-    padding: 4,
-    backgroundColor: '#FFF4DB',
-    fontSize: '1.25rem',
-    borderRadius: 4,
-};
+    line-height: 3.75rem;
+    max-width: 58rem;
+`;
+
+const StyledP = styled.p`
+    ${tw`text-2xl mb-4`}
+
+    line-height: 150%;
+`;
 
 // markup
-const NotFoundPage = () => {
+export default function NotFoundPage({ data }) {
+    const { cta, fourohfour, hero } = data;
     return (
-        <Main>
-            <title>Not found</title>
-            <h1 style={headingStyles}>Page not found</h1>
-            <p style={paragraphStyles}>
-                Sorry{' '}
-                <span role="img" aria-label="Pensive emoji">
-                    😔
-                </span>{' '}
-                we couldn’t find what you were looking for.
-                <br />
-                {process.env.NODE_ENV === 'development' ? (
-                    <>
-                        <br />
-                        Try creating a page in{' '}
-                        <code style={codeStyles}>src/pages/</code>.
-                        <br />
-                    </>
-                ) : null}
-                <br />
-                <Link to="/">Go home</Link>.
-            </p>
-        </Main>
+        <Layout>
+            <Hero content={hero} />
+            <Wrapper>
+                <Img
+                    fixed={fourohfour.childImageSharp.fixed}
+                    style={{ marginTop: '-75px' }}
+                />
+                <StyledHeading>
+                    Didn’t find what you’re looking for?
+                </StyledHeading>
+                <StyledP>
+                    This page doesn’t exist, click <Link to="/">here</Link> to
```

Comment:
The designs show `click here` being wrapped in the `<Link>` component. They're also green and underlined. Convert the `<Link>` to a styled component and use tailwind classes to style.